wire-library/docs/wire_compiler.md

```diff
@@ -215,6 +215,46 @@ The source path and proto path are linked together but only types on the source
 
 ![Library](images/gradle_library@2x.png)
 
+Dependencies between Gradle Modules
+-----------------------------
+
+Wire provides supports to define dependencies between modules within the same project.
+
+A module can include its `.proto` files into the output resources. One would use this when their
+`.jar` file can be used as a library for other proto or Wire projects. Note that only the `.proto`
+files used in the library will be included, and these files will have tree-shaking applied.
```

Comment:
Maybe omit the “... and these” section, since tree shaking isn’t interesting for library dependencies